components/bank/form.js

```diff
@@ -126,15 +127,17 @@ export default function BankApplyForm() {
             <option value="" selected disabled>
               Choose a country
             </option>
-            {countries.countries.map(country => (
-              <option value={country} key={country}>
-                {country}
-              </option>
-            ))}
+            <option value="United States">United States (US)</option>
+            <option value="Canada">Canada (CA)</option>
           </Select>
           <HelperText>
-            We're testing out limited support for international organizations,
-            and want to know in advance if you're operating outside the US.
+            We're currently only able to support projects operating out of the
```

Comment:
Personally, I find this confusing; Bank is "only able to support projects operating out of the United States or Canada", but I "might be eligible to run on Bank" if I'm outside North America?

Reply:
we will not accept applications outside of the US by default, but we still want a way for people to be able to reach out to us. I wasn't really sure how it should be worded though. 

@cjdenio do you have any suggestions?

Reply:
What about smth like this?

We mostly support projects operating out of the United States or Canada but if you're outside of those countries, and might be eligible to run on Bank, please shoot us an...